### PR TITLE
Allow nativefier to set process.env variables

### PR DESCRIPTION
**Pull request by githubbob42**
Fixes issue #418

### CODE REVIEW

File: app/src/main.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ electronDownload();
 const APP_ARGS_FILE_PATH = path.join(__dirname, '..', 'nativefier.json');
 const appArgs = JSON.parse(fs.readFileSync(APP_ARGS_FILE_PATH, 'utf8'));
 
+if (appArgs.processEnvs) {
+  Object.keys(appArgs.processEnvs).forEach((key) => {
+    // eslint-disable-next-line no-undef
```
Comment (ronjouch) [inline app/src/main.js]: @githubbob42 why is `eslint` shouting at you on this line? `process`? If yes, I think [specifying the `node` environment](http://eslint.org/docs/user-guide/configuring#specifying-environments), should make it happy.

Comment (githubbob42) [inline app/src/main.js]: Should I put it in the `.eslintrc.yml` file or just inline it (`/* eslint-env node */`)?

Comment (ronjouch) [inline app/src/main.js]: @githubbob42 I was about to say `.eslintrc`, but our project runs under *two* environments: both node (the builder), and browser (the `app` folder). So, yeah, for now your file-global `eslint-env` looks 👍. Thanks!

**Comment by ronjouch** [comment]
@githubbob42 and one last thing in addition to removing `package-lock.json`: merge of https://github.com/jiahaog/nativefier/pull/244 caused conflicts, can you rebase? Thanks again!

**Comment by githubbob42** [comment]
Hi @ronjouch, rebase it done.  However, should I change the kebab-case of `app-version`, etc (from #244) to camelCase?  It looks like that was done for the other parameters at some point.  Thanks.

**Comment by ronjouch** [comment]
> Hi @ronjouch, rebase it done.  However, should I change the kebab-case of `app-version`, etc (from #244) to camelCase?  It looks like that was done for the other parameters at some point.  Thanks.

@githubbob42 nicely observed 👍. Yas, please.

**Comment by ronjouch** [comment]
Fantasbulous @githubbob42, thanks for the last fixes 👍.

**Pull request merged**